churaverse-plugins-client/src/gamePlugin/domain/baseGamePlugin.ts

```diff
@@ -0,0 +1,49 @@
+import { BasePlugin, IMainScene, Store, IEventBus } from 'churaverse-engine-client'
+import { GameIds } from '../interface/gameIds'
+
+export abstract class BaseGamePlugin extends BasePlugin<IMainScene> {
+  /** ゲーム一意のid */
+  protected readonly id: GameIds
+  /** ゲーム名 */
+  private readonly _name: string
+  /** ゲーム参加者のプレイヤーid */
+  private readonly _participateIds: string[]
+  /** プレイヤーが途中参加かどうかを示すフラグ */
```

Comment:
配列でないのを見るに自プレイヤーが途中参加かどうか、というフラグですかね？
それがわかるようにコメントを変更すると良いかと思います
```suggestion
  /** 自プレイヤーが途中参加かどうかを示すフラグ */
```